src/program/snabbnfv/neutron2snabb/neutron2snabb.lua

```diff
@@ -96,8 +96,13 @@ function filter (port, secbindings, secrules, direction)
          end
       end
    end
-   if #rules > 0 then return { rules = rules }
-                 else return nil end
+   if type == "stateless" then
```

Comment:
OK.

We need a way to disable security group filtering entirely for a port. Do you know a suitable one?